src/site/_includes/components/navigation/macro.njk

```diff
@@ -0,0 +1,15 @@
+{% macro navigation(params) %}
```

Comment:
Just had a look at test/eleventy-page-list branch I will update this to use `collection.all`